bin/ofs/Cargo.toml

```diff
@@ -52,3 +51,8 @@ bytes = "1.5.0"
 libc = "0.2.151"
 fuse3 = { "version" = "0.6.1", "features" = ["tokio-runtime", "unprivileged"] }
 nix = { version = "0.27.1", features = ["user"] }
+
+[features]
+default = ["fs-backend", "s3-backend"]
+fs-backend = ["opendal/services-fs"]
```

Comment:
How about following the same feature names like opendal?

Reply:
> implement test/release workflows easier

We have an integration test framework within [test_behavior](https://github.com/apache/opendal/blob/main/.github/workflows/test_behavior.yml) that relies on feature names. All existing packages adhere to this pattern. Different feature patterns make it harder for us to maintain workflows.

Would you like to reconsider this?

Reply:
Thanks a lot for your understanding!